src/ui/prompts/listPrompts/variationsListPrompt.ts

```diff
@@ -16,19 +19,74 @@ export class VariationListOptions extends ListOptionsPrompt<CreateVariationParam
             ExitPrompt
         ]
     }
+
+    featureVariables: Variable[] = []
+
+    getVariationListPrompt = (existingVariations?: Variation[], existingVariables?: Variable[]) => {
+        this.featureVariables = existingVariables || []
+
+        return {
+            name: 'variations', 
+            value: 'variations', 
+            message: 'Manage variations',
+            type: 'listOptions',
+            previousReponseFields: ['variables'], // if variables were just created / updated, use the new variables
+            listOptionsPrompt: (previousResponses?: Record<string, any>) => {
+                const newVariables = previousResponses?.variables
+                this.featureVariables = newVariables || []
+
+                return this.prompt(existingVariations?.map((variation) => ({
+                    name: variation.name || variation.key,
+                    value: variation
+                })))
+            }
+        }
+    }
+
     async promptAddItem<CreateVariationParams>(): Promise<ListOption<CreateVariationParams>> {
-        // TODO: Implement add item
+        const variation = await inquirer.prompt(staticCreateVariationPrompts)
+        variation.variables = await inquirer.prompt(getVariationVariablesPrompts(this.featureVariables))
+
         return {
-            name: 'Placeholder',
-            value: {} as CreateVariationParams
+            name: variation.name,
+            value: variation as CreateVariationParams
         }
     }
 
     async promptEditItem<CreateVariationParams>(
         list: ListOption<CreateVariationParams>[]
-    ): Promise<ListOption<CreateVariationParams>> {
-        // TODO: Implement edit item
-        return Promise.resolve(list[0])
+    ): Promise<void> {
+        if (list.length === 0) {
+            this.writer.warningMessage('No variations to edit')
+            return
+        }
+        const response = await inquirer.prompt([{
+            name: 'variationToEdit',
+            message: 'Which variation would you like to edit?',
+            type: 'list',
+            choices: list
+        }])
+        const index = list.findIndex((listItem) => listItem.name === response.variationToEdit.name)
```

Comment:
This could potentially be problematic since I believe variation names are not necessarily unique. A potential workaround would be to include in the index as part of the response like so ```{
	name: "string",
	value: { variation, index }
}```
but id be interested to hear your thoughts

Reply:
yeah our UX for 2 variations with the same name is pretty bad since we only show the name when you select from a list

we can at least make sure we update the right one by using key instead of name like 
      `const index = list.findIndex((listItem) => (listItem.value as Variation).key === response.variationToEdit.key)`
since we have access to the full variation in the list items 

(not sure why i can't get the type to work, since listItem.value should be a CreateVariationParams type, but it keeps insisting key doesn't exist)

Reply:
oh wait never mind figured out the types issue, it wasn't passing the real `CreateVariationParams`, it was just an alias for the type passed to `promptEditItem`. I think I fixed it now

Reply:
cool, that seems like a good option